frontend/apps/web/components/RunTImeline/RunTimeline.tsx

```diff
@@ -56,9 +61,12 @@ export default function RunTimeline(props: Props): ReactElement {
     let endTime = -Infinity;
 
     tasks.forEach((t) => {
+      const scheduled = t.tasks.find(
+        (st) => st.type == 'ActivityTaskScheduled'
+      )?.eventTime;
       startTime = Math.min(
         startTime,
-        convertTimestampToDate(t.startTime).getTime()
+        convertTimestampToDate(scheduled).getTime()
```

Comment:
Hm, can you explain why there is a specific edge case here for the scheduled task?

Reply:
there isn't so much of an edge case but i did it to match up with the metadata in the screenshot.

for ex. this is how it was before

![image](https://github.com/user-attachments/assets/bad2b3d4-3619-414a-9f4c-2b84cee37381)

and this is it after the change:

![image](https://github.com/user-attachments/assets/2442594c-fc85-4815-a68a-df48fd95d04d)

in the before, the  scheduled -> start -> completed doesn't match up with the label on the timeline bar above "50" but that doesn't account for the time from when it was scheduled to when it started. so it looks like a little weird since in the metadata details its technically 53 seconds including the time to schedule

in the after, the scheduled -> start -> completed follows the label on the timeline bar above "53" ms since that label is calculating the duration from scheduled -> close. 

we could also just update the timeline label and leave the metadata as in the before it was more of a judgment call where i felt like in the update it accounted for the time from scheduled -> start and in the before it didn't and i felt like it was more inclusive. 

open to suggestions
scheduled: 
start: